prow/gerrit/reporter/reporter_test.go

```diff
@@ -614,7 +797,7 @@ func TestReport(t *testing.T) {
 			continue
 		}
 
-		err := reporter.Report(tc.pj)
+		_, err := reporter.Report(tc.pj)
```

Comment:
I'll suggest also check return values of each tests.

Also I think you want to add a test case for behavior like {A1(reported), A2(new), B1(reported)}

Reply:
done